naucse/templates/_base.html

```diff
@@ -13,7 +13,7 @@
 
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
 
-        {% if page is defined and page.latex %}
+        {% if page is defined and page.latex is defined and page.latex %}
```

Comment:
Can we use a different name than `page` for the cover pages, so that `page` is reserved only for `Page` objects?